angular/lib/src/core/linker/app_view.dart

```diff
@@ -584,7 +583,7 @@ abstract class AppView<T> {
     domRootRendererIsDirty = true;
   }
 
-  VoidFunc1<E> eventHandler0<E>(VoidFunc0 handler) {
+  void Function(E) eventHandler0<E>(void Function() handler) {
```

Comment:
In that case there are some old style function parameters in these files you could update as well.